src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -21806,10 +21724,90 @@ public RefByteContainer Create(ref ByteContainer bc)
                 """;
 
             var comp = CreateCompilation(source, runtimeFeature: RuntimeFlag.ByRefFields);
+            comp.VerifyDiagnostics();
+        }
+
+        [Fact]
+        public void OutReturnOnly_Ctor1()
```

Comment:
>OutReturnOnly_Ctor1 [](http://example.com/codeflow?start=20&length=19)

What scenario are we testing here?

Reply:
Added a comment

Reply:
There's no difference in behavior with this test in this PR, correct?